cpp/test/Ice/slicing/objects/TestI.cpp

```diff
@@ -26,67 +24,67 @@ void breakCycles(const vector<shared_ptr<T>>& s)
 }
 
 template<typename K, typename V>
-void breakCycles(const map<K, shared_ptr<V>>& d)
+void breakCycles(const map<K, ICE_SHARED_PTR<V>>& d)
```

Comment:
Can we define SharedPtr as an alias for shared_ptr in the new mapping, to avoid using this macro?

Reply:
No, in this case, macros are much better.

The ultimate goal here is to have a single library for both mappings. The symbol `Ice::SharedPtr<T>` must have a single meaning in this library. When we reach this point, the implementation of libIce and any other library that supports both mappings will not use any of these helper macros. We should actually move these helper macros to the test suite and remove them from Ice/IceUtil.